pluribus/games/short_deck/state.py

```diff
@@ -173,15 +175,16 @@ def __init__(
         self._initial_n_chips = players[0].n_chips
         self.small_blind = small_blind
         self.big_blind = big_blind
-        self.real_time = real_time
+        self.real_time_test = real_time_test
         self._poker_engine = PokerEngine(
             table=self._table, small_blind=small_blind, big_blind=big_blind
         )
         # Reset the pot, assign betting order to players (might need to remove
         # this), assign blinds to the players.
         self._poker_engine.round_setup()
         # Deal private cards to players.
-        self._table.dealer.deal_private_cards(self._table.players)
+        if not self.real_time_test:
```

Comment:
During real time play, we'd actually want this deal to happen when the new game state is instantiated. For the test, I'm just keeping things simple, since we won't need to deal hole cards until the test node is reached. 

There is a better way to abstract this, but I'm just focusing on getting something working